src/mysky.ts

```diff
@@ -1,5 +1,16 @@
+/**
+ * @file Contains the interface for Main MySky, the MySky component that lives
+ * in an invisible iframe on skapps.
+ *
+ * TODO: Document when Main MySky is launched and what the entrypoint in this
```

Comment:
important? looks like it might be nice to do these TODOs inside this PR, if not let's create an issue for it

Reply:
I'd vote for F/Us. This PR is large enough and I'm already struggling to follow.